src/tink/http/Header.hx

```diff
@@ -62,6 +62,7 @@ class Header {
     return byName('content-type').map(ContentType.ofString);
 }
 
+@:forward
 abstract HeaderValue(String) from String to String {
```

Comment:
This is mainly due to #13 that I need to `split` the header value. However if #14 is merged the problem will be solved and I have no immediate need for this anymore.